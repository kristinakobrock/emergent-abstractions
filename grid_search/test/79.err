  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  4%|###7                                                                                         | 2/50 [00:00<00:02, 18.51it/s] 10%|#########3                                                                                   | 5/50 [00:00<00:01, 25.05it/s] 16%|##############8                                                                              | 8/50 [00:00<00:01, 26.88it/s] 22%|####################2                                                                       | 11/50 [00:00<00:01, 24.63it/s] 28%|#########################7                                                                  | 14/50 [00:00<00:01, 22.18it/s] 34%|###############################2                                                            | 17/50 [00:00<00:01, 23.64it/s] 40%|####################################8                                                       | 20/50 [00:00<00:01, 22.19it/s] 46%|##########################################3                                                 | 23/50 [00:00<00:01, 23.17it/s] 52%|###############################################8                                            | 26/50 [00:01<00:01, 23.67it/s] 58%|#####################################################3                                      | 29/50 [00:01<00:00, 25.19it/s] 64%|##########################################################8                                 | 32/50 [00:01<00:00, 25.26it/s] 72%|##################################################################2                         | 36/50 [00:01<00:00, 26.58it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 25.30it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 21.86it/s] 90%|##################################################################################8         | 45/50 [00:01<00:00, 22.40it/s] 96%|########################################################################################3   | 48/50 [00:02<00:00, 23.97it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.04it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 24.96it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 23.07it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 25.50it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 23.88it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.22it/s]
