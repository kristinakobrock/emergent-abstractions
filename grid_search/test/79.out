# ["--attributes=5", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6969271302223206, "acc": 0.5927801132202148, "length": 1.9171615839004517, "mode": "train", "epoch": 1}
{"loss": 0.6723151803016663, "acc": 0.5951171517372131, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6623278260231018, "acc": 0.6243881583213806, "length": 1.0719393491744995, "mode": "train", "epoch": 2}
{"loss": 0.6630507111549377, "acc": 0.6236328482627869, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6600388884544373, "acc": 0.6243533492088318, "length": 1.0564426183700562, "mode": "train", "epoch": 3}
{"loss": 0.6629899740219116, "acc": 0.6251952648162842, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6602280735969543, "acc": 0.6257484555244446, "length": 1.040485143661499, "mode": "train", "epoch": 4}
{"loss": 0.662079930305481, "acc": 0.6259765625, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6597853899002075, "acc": 0.6237697005271912, "length": 1.0356266498565674, "mode": "train", "epoch": 5}
{"loss": 0.6674337387084961, "acc": 0.626757800579071, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6588879823684692, "acc": 0.6255685687065125, "length": 1.0264283418655396, "mode": "train", "epoch": 6}
{"loss": 0.6649011969566345, "acc": 0.6259765028953552, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6591044664382935, "acc": 0.6250914335250854, "length": 1.024225115776062, "mode": "train", "epoch": 7}
{"loss": 0.6636897325515747, "acc": 0.623828113079071, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6598212122917175, "acc": 0.6240924596786499, "length": 1.0246694087982178, "mode": "train", "epoch": 8}
{"loss": 0.6661733984947205, "acc": 0.6224609017372131, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6602141857147217, "acc": 0.6245527267456055, "length": 1.027314305305481, "mode": "train", "epoch": 9}
{"loss": 0.6618009805679321, "acc": 0.6265625357627869, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6578394770622253, "acc": 0.624832272529602, "length": 1.022721767425537, "mode": "train", "epoch": 10}
{"loss": 0.6626938581466675, "acc": 0.6263671517372131, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6592174768447876, "acc": 0.6233178973197937, "length": 1.0296096801757812, "mode": "train", "epoch": 11}
{"loss": 0.6684313416481018, "acc": 0.627148449420929, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.6589537858963013, "acc": 0.6267424821853638, "length": 1.032503604888916, "mode": "train", "epoch": 12}
{"loss": 0.6711028218269348, "acc": 0.5980468392372131, "length": 1.0, "mode": "test", "epoch": 12}
{"loss": 0.6424216032028198, "acc": 0.6395338177680969, "length": 1.5894745588302612, "mode": "train", "epoch": 13}
{"loss": 0.6607140302658081, "acc": 0.628527820110321, "length": 1.8984375, "mode": "test", "epoch": 13}
{"loss": 0.6167750954627991, "acc": 0.6718194484710693, "length": 1.6695542335510254, "mode": "train", "epoch": 14}
{"loss": 0.6682620048522949, "acc": 0.612353503704071, "length": 1.58984375, "mode": "test", "epoch": 14}
{"loss": 0.595421552658081, "acc": 0.6924086213111877, "length": 2.1510889530181885, "mode": "train", "epoch": 15}
{"loss": 0.5977921485900879, "acc": 0.695050060749054, "length": 3.1328125, "mode": "test", "epoch": 15}
{"loss": 0.5721562504768372, "acc": 0.7091936469078064, "length": 2.8141517639160156, "mode": "train", "epoch": 16}
{"loss": 0.5944417119026184, "acc": 0.6934021711349487, "length": 3.1796875, "mode": "test", "epoch": 16}
{"loss": 0.553408682346344, "acc": 0.7262430191040039, "length": 2.9181017875671387, "mode": "train", "epoch": 17}
{"loss": 0.5685388445854187, "acc": 0.7134033441543579, "length": 3.19140625, "mode": "test", "epoch": 17}
{"loss": 0.5269909501075745, "acc": 0.7438687086105347, "length": 2.9822940826416016, "mode": "train", "epoch": 18}
{"loss": 0.5696180462837219, "acc": 0.7273010611534119, "length": 3.23828125, "mode": "test", "epoch": 18}
{"loss": 0.5072225332260132, "acc": 0.7589656710624695, "length": 3.061417579650879, "mode": "train", "epoch": 19}
{"loss": 0.5668675899505615, "acc": 0.7329467535018921, "length": 3.2734375, "mode": "test", "epoch": 19}
{"loss": 0.4825369417667389, "acc": 0.776315450668335, "length": 3.1253395080566406, "mode": "train", "epoch": 20}
{"loss": 0.5354528427124023, "acc": 0.7568848133087158, "length": 3.33203125, "mode": "test", "epoch": 20}
{"loss": 0.46533989906311035, "acc": 0.7867063283920288, "length": 3.194524049758911, "mode": "train", "epoch": 21}
{"loss": 0.5155714750289917, "acc": 0.753741443157196, "length": 3.203125, "mode": "test", "epoch": 21}
{"loss": 0.43450871109962463, "acc": 0.8053230047225952, "length": 3.2767319679260254, "mode": "train", "epoch": 22}
{"loss": 0.4769998788833618, "acc": 0.7850036025047302, "length": 3.5546875, "mode": "test", "epoch": 22}
{"loss": 0.3987461030483246, "acc": 0.823438286781311, "length": 3.6603918075561523, "mode": "train", "epoch": 23}
{"loss": 0.501884400844574, "acc": 0.7736329436302185, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.36566850543022156, "acc": 0.8399076461791992, "length": 3.9445109367370605, "mode": "train", "epoch": 24}
{"loss": 0.4438236653804779, "acc": 0.8236327767372131, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.34040603041648865, "acc": 0.857467532157898, "length": 3.9879226684570312, "mode": "train", "epoch": 25}
{"loss": 0.4232707619667053, "acc": 0.8291015625, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.3117496967315674, "acc": 0.874138355255127, "length": 3.989074945449829, "mode": "train", "epoch": 26}
{"loss": 0.40766435861587524, "acc": 0.8328124284744263, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.2890549302101135, "acc": 0.8869324922561646, "length": 3.993177890777588, "mode": "train", "epoch": 27}
{"loss": 0.42740103602409363, "acc": 0.8265624046325684, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.27835163474082947, "acc": 0.8921292424201965, "length": 3.9940707683563232, "mode": "train", "epoch": 28}
{"loss": 0.4245036244392395, "acc": 0.8330078721046448, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.2544948160648346, "acc": 0.904970109462738, "length": 3.9956727027893066, "mode": "train", "epoch": 29}
{"loss": 0.37047886848449707, "acc": 0.8644531965255737, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.24053002893924713, "acc": 0.9100876450538635, "length": 3.9963552951812744, "mode": "train", "epoch": 30}
{"loss": 0.3685626983642578, "acc": 0.8642578125, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.23081931471824646, "acc": 0.9160106182098389, "length": 3.996990919113159, "mode": "train", "epoch": 31}
{"loss": 0.3689008951187134, "acc": 0.8664063215255737, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.22417297959327698, "acc": 0.9178778529167175, "length": 3.996467113494873, "mode": "train", "epoch": 32}
{"loss": 0.3863091766834259, "acc": 0.8603515625, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.2082774043083191, "acc": 0.9272043704986572, "length": 3.997462272644043, "mode": "train", "epoch": 33}
{"loss": 0.364425390958786, "acc": 0.8763671517372131, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.1974802017211914, "acc": 0.9336454272270203, "length": 3.9976789951324463, "mode": "train", "epoch": 34}
{"loss": 0.3504056930541992, "acc": 0.8757811188697815, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.18547488749027252, "acc": 0.9377074241638184, "length": 3.9980885982513428, "mode": "train", "epoch": 35}
{"loss": 0.3516809344291687, "acc": 0.8876952528953552, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.19360829889774323, "acc": 0.9332095384597778, "length": 3.998548984527588, "mode": "train", "epoch": 36}
{"loss": 0.3798985779285431, "acc": 0.8845702409744263, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.18137438595294952, "acc": 0.9386328458786011, "length": 3.9978342056274414, "mode": "train", "epoch": 37}
{"loss": 0.3280474841594696, "acc": 0.8941406607627869, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.1857447475194931, "acc": 0.9359650015830994, "length": 3.998593807220459, "mode": "train", "epoch": 38}
{"loss": 0.349221408367157, "acc": 0.8904296159744263, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.17455235123634338, "acc": 0.9401242733001709, "length": 3.998802423477173, "mode": "train", "epoch": 39}
{"loss": 0.3366325795650482, "acc": 0.8962889909744263, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.1607976257801056, "acc": 0.9463430047035217, "length": 3.9990501403808594, "mode": "train", "epoch": 40}
{"loss": 0.3253892660140991, "acc": 0.9060546159744263, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.15955011546611786, "acc": 0.9477405548095703, "length": 3.999025344848633, "mode": "train", "epoch": 41}
{"loss": 0.3479689359664917, "acc": 0.8980469107627869, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.1607431322336197, "acc": 0.9472757577896118, "length": 3.999082088470459, "mode": "train", "epoch": 42}
{"loss": 0.32254600524902344, "acc": 0.9074219465255737, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.1544937938451767, "acc": 0.9488668441772461, "length": 3.999312162399292, "mode": "train", "epoch": 43}
{"loss": 0.31884580850601196, "acc": 0.9025391340255737, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.14661529660224915, "acc": 0.9511544108390808, "length": 3.9995839595794678, "mode": "train", "epoch": 44}
{"loss": 0.3367966413497925, "acc": 0.9013672471046448, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.146455779671669, "acc": 0.9507452249526978, "length": 3.999544620513916, "mode": "train", "epoch": 45}
{"loss": 0.3170885145664215, "acc": 0.912109375, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.1463533639907837, "acc": 0.9526203274726868, "length": 3.999560832977295, "mode": "train", "epoch": 46}
{"loss": 0.3295012414455414, "acc": 0.90625, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.1405695229768753, "acc": 0.9535568356513977, "length": 3.999635696411133, "mode": "train", "epoch": 47}
{"loss": 0.33421799540519714, "acc": 0.9046874046325684, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.13706639409065247, "acc": 0.9538544416427612, "length": 3.9997003078460693, "mode": "train", "epoch": 48}
{"loss": 0.31463462114334106, "acc": 0.9093750715255737, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.14153993129730225, "acc": 0.9527941942214966, "length": 3.9995360374450684, "mode": "train", "epoch": 49}
{"loss": 0.3263518512248993, "acc": 0.90625, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.13855604827404022, "acc": 0.9542651772499084, "length": 3.9997267723083496, "mode": "train", "epoch": 50}
{"loss": 0.3295169174671173, "acc": 0.9076171517372131, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.13523732125759125, "acc": 0.9545603394508362, "length": 3.999800205230713, "mode": "train", "epoch": 51}
{"loss": 0.3102850615978241, "acc": 0.9103516340255737, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.13192039728164673, "acc": 0.9564902782440186, "length": 3.999793767929077, "mode": "train", "epoch": 52}
{"loss": 0.3348870575428009, "acc": 0.9087890982627869, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.13308511674404144, "acc": 0.9546186923980713, "length": 3.9998109340667725, "mode": "train", "epoch": 53}
{"loss": 0.3113481402397156, "acc": 0.9124999642372131, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.13761569559574127, "acc": 0.9536246657371521, "length": 3.9998362064361572, "mode": "train", "epoch": 54}
{"loss": 0.3297376036643982, "acc": 0.9117187857627869, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.13738805055618286, "acc": 0.9536240100860596, "length": 3.9998321533203125, "mode": "train", "epoch": 55}
{"loss": 0.28561097383499146, "acc": 0.9189453125, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.12307476252317429, "acc": 0.958543062210083, "length": 3.999851703643799, "mode": "train", "epoch": 56}
{"loss": 0.32564133405685425, "acc": 0.9095702767372131, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.12470695376396179, "acc": 0.9571383595466614, "length": 3.999858856201172, "mode": "train", "epoch": 57}
{"loss": 0.3260529041290283, "acc": 0.9124999642372131, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.12521551549434662, "acc": 0.9559057950973511, "length": 3.999758720397949, "mode": "train", "epoch": 58}
{"loss": 0.32468515634536743, "acc": 0.9091796875, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.11967409402132034, "acc": 0.9590691328048706, "length": 3.9997799396514893, "mode": "train", "epoch": 59}
{"loss": 0.3074421286582947, "acc": 0.9193359017372131, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.11776051670312881, "acc": 0.959129810333252, "length": 3.999903678894043, "mode": "train", "epoch": 60}
{"loss": 0.2848566472530365, "acc": 0.9232421517372131, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.8251667022705078
